### PR TITLE
feat: updated a11y component

### PR DESCRIPTION
**Pull request by liamross**
1. Simplified a11y component, added jump to search
1. Removed hotkey enter listeners

TO TEST:

1. Run accessibility sample
   - Should have helper to skip to sections when you tab into it
   - Should skip to sections correctly
1. Notes on annotations should be focusable
   - Pressing enter to select a note when tabbed to it should not insert a line break into the comment field

### CODE REVIEW

File: src/helpers/hotkeysManager.js [hunk 1]
```diff
@@ -330,19 +330,6 @@ WebViewer(...)
...
-        } else if (document.activeElement.className === 'skip-to-document') {
-          document.getElementById('pageText0').focus();
-        } else if (document.activeElement.className === 'skip-to-notes') {
-          dispatch(actions.openElement('notesPanel'));
-          const noteEl = document.querySelector('.Note');
-          if (noteEl) {
-            noteEl.focus();
-          }
-        }
-      },
```
Comment (liamross) [inline src/helpers/hotkeysManager.js:345]: This section wasn't needed. Notes are now focusable and keyboard clickable, and the a11y block elements have their own on-click functions.

File: src/components/Note/Note.js [hunk 2]
```diff
@@ -123,7 +124,14 @@ const Note = ({ annotation }) => {
   );
 
   return (
-    <div ref={containerRef} className={noteClass} onClick={handleNoteClick} onKeyDown={handleNoteKeydown}>
+    <div
+      role="button"
+      tabIndex={0}
```
Comment (liamross) [inline src/components/Note/Note.js:129]: focusable and with a role

File: src/components/Accessibility/Accessibility.scss [hunk 3]
```diff
@@ -2,37 +2,33 @@
 
 .Accessibility {
   position: fixed;
-  left: 50%;
-  transform: translate(-50%);
-  z-index: 30;
+  z-index: 9999;
```
Comment (liamross) [inline src/components/Accessibility/Accessibility.scss:5]: 30 wasn't enough to put it on top of webviewer

**Comment by ZhijieZhang** [comment]
I tested and it worked well. Can you take a look at the code in the Escape hotkey? I feel like it's dead.

If I remember correctly the original functionality is that in the a11y mode, when we hit `escape`, the helper will pop up and users can then tab to choose to skip to document or the notes panel.

However that code seems to be dead or not functioning for a long time, so what you have now is great. You can merge it in if Kristian is busy.

**Pull request merged**